clean_air/util/file_converter.py

```diff
@@ -0,0 +1,106 @@
+    save as JSON string.
+    """
+    # iterate through each row to split into separate dataframes and save in
+    # clean_air/data/json_data/:
+    for r, row in enumerate(temp_xl_df.iterrows()):
+        print('extracting data from row ', r, ' right now...')
+
+        # obtain data from excel file:
+        self.title = row[1].values[16]  # title of the model, not the user.
+        self.description = row[1].values[17]  # again, description of model.
+        # self.cs = ?
+        self.firstname1 = row[1].values[6]
+        self.surname1 = row[1].Surname
+        self.firstname2 = row[1].values[11]
+        self.surname2 = row[1].values[12]
+        self.north = row[1].values[42]
+        self.south = row[1].values[41]
+        self.east = row[1].values[43]
+        self.west = row[1].values[44]
+        self.chemicals = row[1].values[46]
+        self.obs_level = row[1].values[19]
+        self.data_source = row[1].values[20]
+        self.time_range_start = row[1].values[37]
+        self.time_range_end = row[1].values[38]
+        self.lineage = row[1].values[50]
+        self.quality = row[1].values[51]
+        self.docs = row[1].values[22]
+
+        # turn this into an OrderedDict first, then save to json:
+        new_file = OrderedDict({"title": self.title,
+                                "description": self.description,
+                                "authors":
+                                    ["firstname", self.firstname1,
+                                     "surname", self.surname1,
+                                     "firstname2", self.firstname2,
+                                     "surname2", self.surname2],
+                                "bbox":
+                                    ["north", self.north,
+                                     "south", self.south,
+                                     "east", self.east,
+                                     "west", self.west],
+                                "chemical species": self.chemicals,
+                                "observation level/model": self.obs_level,
+                                "data source": self.data_source,
+                                "time range":
+                                    ["start", self.time_range_start,
+                                     "end", self.time_range_end],
+                                "lineage": self.lineage,
+                                "quality": self.quality,
+                                "docs": self.docs})
+
+        # subclass JSONEncoder
+        class DateTimeEncoder(JSONEncoder):
+            # Override default method so we can extract and encode datetimes:
+            def default(self, obj):
+                if isinstance(obj, (datetime.date, datetime.datetime)):
+                    return obj.isoformat()
+
+        with open('../data/json_data/dailydata' + str(r) + '.json', 'w') as fp:
+            json.dump(new_file, fp, indent=0, cls=DateTimeEncoder)
```

Comment:
An indent greater than 0 would make the output more human-readable

Reply:
What do you fancy then, 2, 4?

Reply:
It doesn't matter in the end but I tend to use 2 spaces in json files